api-report/migration-shim.api.md

```diff
@@ -18,6 +18,8 @@ import { ISharedTree } from '@fluid-experimental/tree2';
 import { ISummaryTreeWithStats } from '@fluidframework/runtime-definitions';
 import { ITelemetryContext } from '@fluidframework/runtime-definitions';
 import { SharedTree } from '@fluid-experimental/tree';
+import { SharedTreeFactory } from '@fluid-experimental/tree';
+import { SharedTreeFactory as SharedTreeFactory_2 } from '@fluid-experimental/tree2';
```

Comment:
Note: this is the API extractor md file, in code the first `SharedTreeFactory` is marked as `LegacySharedTreeFactory` and this is `SharedTreeFactory_2` is `SharedTreeFactory`